plugins/com.redhat.fabric8analytics.lsp.eclipse.ui/src/com/redhat/fabric8analytics/lsp/eclipse/ui/ExitHandler.java

```diff
@@ -61,15 +61,11 @@ public Object execute(ExecutionEvent event) throws ExecutionException {
 				MessageDialogUtils.displayInfoMessage("Enable Fabric8 Analyses");
 				return null;
 			}
-			String token = Fabric8AnalysisPreferences.getInstance().getToken();
+			String token = TokenCheck.getInstance().getToken();
```

Comment:
This will open the login page, which we do not want until analyses is enabled in preferences.

Reply:
Actually there is a check if analyses are enabled in preferences https://github.com/rawagner/fabric8-analytics-devstudio-plugin/blob/84b400766cf217a8a5e77f05cc5642cb0f67bd30/plugins/com.redhat.fabric8analytics.lsp.eclipse.ui/src/com/redhat/fabric8analytics/lsp/eclipse/ui/ExitHandler.java#L60